rmf_site_editor/src/widgets/inspector/inspect_drawing.rs

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2024 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+use crate::{
+    site::{AlignSiteDrawings, BeginEditDrawing, Change, PixelsPerMeter},
+    widgets::{prelude::*, Inspect, InspectValue},
+    AppState, CurrentWorkspace, Icons,
+};
+use bevy::prelude::*;
+use bevy_egui::egui::Button;
+
+#[derive(SystemParam)]
+pub struct InspectDrawing<'w, 's> {
+    pixels_per_meter: Query<'w, 's, &'static PixelsPerMeter>,
+    change_pixels_per_meter: EventWriter<'w, Change<PixelsPerMeter>>,
+    current_workspace: Res<'w, CurrentWorkspace>,
+    align_site: EventWriter<'w, AlignSiteDrawings>,
+    app_state: Res<'w, State<AppState>>,
+    icons: Res<'w, Icons>,
+    begin_edit_drawing: EventWriter<'w, BeginEditDrawing>,
+}
+
+impl<'w, 's> WidgetSystem<Inspect> for InspectDrawing<'w, 's> {
+    fn show(
+        Inspect { selection, .. }: Inspect,
+        ui: &mut Ui,
+        state: &mut SystemState<Self>,
+        world: &mut World,
+    ) {
+        let mut params = state.get_mut(world);
+        let Ok(ppm) = params.pixels_per_meter.get(selection) else {
+            return;
+        };
+
+        if *params.app_state.get() == AppState::SiteEditor {
+            ui.add_space(10.0);
+            if ui
+                .add(Button::image_and_text(
+                    params.icons.edit.egui(),
+                    "Edit Drawing",
+                ))
+                .clicked()
+            {
+                params.begin_edit_drawing.send(BeginEditDrawing(selection));
+            }
+        }
+        ui.add_space(10.0);
+
+        if ui
+            .add(Button::image_and_text(
+                params.icons.alignment.egui(),
+                "Align Drawings",
+            ))
+            .on_hover_text(
+                "Align all drawings in the site based on their fiducials and measurements",
+            )
+            .clicked()
+        {
+            if let Some(site) = params.current_workspace.root {
+                params.align_site.send(AlignSiteDrawings(site));
+            }
```

Comment:
We probably ended up in that position because of the hackery we do of pulling the `Drawing` out of the site tree during drawing editor mode. We could fix this with the new visibility inheritance options that were introduced to bevy since then, but for now I've just removed that button from this mode: [fc8f1a6](https://github.com/open-rmf/rmf_site/pull/229/commits/fc8f1a648e16d597951640c1ea69239354bbdce9)